recipe/meta.yaml

```diff
@@ -24,16 +24,24 @@ requirements:
     - pip
   run:
     - python >=3.6
-    - toml
+    - tomli >=1.1.0
 
 test:
   requires:
     - pip
+    - coverage ==7.0.5
+    - pint
+    - pytest ==7.2.1
+    - pytest-cov ==4.0.0
```

Comment:
Am guessing these constraints come from [`tox.ini`]( https://github.com/jendrikseipp/vulture/blob/e66b203cdcf93a6fae02200db51af6e4bd2a46b3/tox.ini#L12-L17 ). Is that right?

Think they are pinning just from a reproducibility point on their CI. In our case, we can probably relax these, which make this easier to maintain

Reply:
Sure, not an issue. The CI used here is run only once to generate the package, but I'll open then pins.